src/gsi/gsi_obOperTypeManager.F90

```diff
@@ -280,20 +286,18 @@ function dtype2index_(dtype) result(index_)
     case("ompstc8"); index_= iobOper_oz
     case("ompsnp" ); index_= iobOper_oz
     case("ompsnm" ); index_= iobOper_oz
-    case("omieff"   ); index_= iobOper_oz
```

Comment:
Why are there changes in the ozone section of this file?

Reply:
Updated this section to match the Develop branch.

Reply:
Differences remain between `develop` and `KariA-Spire:gnssrwnd1.0` with regards to ozone. 

`develop:gsi_obOperTypeManager.F90` [contains](https://github.com/NOAA-EMC/GSI/blob/59d7578b31454140cb38bf65b27e3cffb02c7e3e/src/gsi/gsi_obOperTypeManager.F90#L276-#L296) 
```
  case("oz"     ,"[ozoper]"     ); index_= iobOper_oz
    case("sbuv2"  ); index_= iobOper_oz
    case("omi"    ); index_= iobOper_oz
    case("gome"   ); index_= iobOper_oz
    case("ompstc8"); index_= iobOper_oz
    case("ompsnp" ); index_= iobOper_oz
    case("ompsnm" ); index_= iobOper_oz
    case("omieff"   ); index_= iobOper_oz
    case("tomseff"  ); index_= iobOper_oz
    case("ompsnmeff"); index_= iobOper_oz

  case("o3l"    ,"[o3loper]"    ); index_= iobOper_o3l
    case("o3lev"    ); index_= iobOper_o3l
    case("mls20"    ); index_= iobOper_o3l
    case("mls22"    ); index_= iobOper_o3l
    case("mls30"    ); index_= iobOper_o3l
    case("mls55"    ); index_= iobOper_o3l
    case("ompslp"   ); index_= iobOper_o3l
    case("ompslpuv" ); index_= iobOper_o3l
    case("ompslpvis"); index_= iobOper_o3l
    case("ompslpnc" ); index_= iobOper_o3l
```
whereas `KariA-Spire:gnssrwnd1.0:gsi_obOperTypeManager` [contains](https://github.com/KariA-Spire/GSI/blob/2aba9bc75fc4ee120e122a7ac0863f4851ee26b0/src/gsi/gsi_obOperTypeManager.F90#L282-#L304)
```
  case("o3l"    ,"[o3loper]"    ); index_= iobOper_o3l
    case("o3lev"    ); index_= iobOper_o3l
    case("mls20"    ); index_= iobOper_o3l
    case("mls22"    ); index_= iobOper_o3l
    case("mls30"    ); index_= iobOper_o3l
    case("mls55"    ); index_= iobOper_o3l
    case("ompslp"   ); index_= iobOper_o3l
    case("ompslpuv" ); index_= iobOper_o3l
    case("ompslpvis"); index_= iobOper_o3l
    case("ompslpnc" ); index_= iobOper_o3l

  case("o3l"    ,"[o3loper]"    ); index_= iobOper_o3l
    case("o3lev"    ); index_= iobOper_o3l
    case("mls20"    ); index_= iobOper_o3l
    case("mls22"    ); index_= iobOper_o3l
    case("mls30"    ); index_= iobOper_o3l
    case("mls55"    ); index_= iobOper_o3l

    case("omieff"   ); index_= iobOper_o3l
    case("tomseff"  ); index_= iobOper_o3l
    case("ompslpuv" ); index_= iobOper_o3l
    case("ompslpvis"); index_= iobOper_o3l
    case("ompslp"   ); index_= iobOper_o3l
```
`KariA-Spire:gnssrwnd1.0:gsi_obOperTypeManager` contains two sections for `case("o3l"`

Reply:
Ozone differences still remain with respect to `develop`

Reply:
Corrected "oz" and "o3loper" blocks to match develop branch